src/Sarif.UnitTests/Processors/GenericTests/GenericProcessorTests.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) Microsoft. All rights reserved. Licensed under the MIT        
+// license. See LICENSE file in the project root for full license information. 
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Xunit;
+using FluentAssertions;
+
+namespace Microsoft.CodeAnalysis.Sarif.Processors
+{
+    public class GenericProcessorTests
+    {
+        private List<int> GenerateRandomIntList()
```

Comment:
>GenerateRandomIntList [](start = 26, length = 21)

Create some API use tests against your extension methods? [NOTE: this comment doesn't apply to this file but to a test class that doesn't exist yet :)]

Reply:
Added a few.